ui_framework/doc_site/src/views/tool_bar/tool_bar.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+
+import {
+  KuiToolBar,
+  KuiToolBarSearchBox,
+  KuiButton,
+  KuiButtonIcon,
+  KuiButtonGroup,
+} from '../../../../components';
+
+export const ToolBar = () => (
+  <KuiToolBar>
+    <KuiToolBarSearchBox onFilter={() => {}}/>
+
+    <div>
+      <select className="kuiSelect">
+        <option>Past hour</option>
+        <option>Past day</option>
+        <option>Past week</option>
+      </select>
+    </div>
+
+    <div className="kuiToolBarSection">
```

Comment:
I agree we should do this, (and the below kuiToolBarText), but because it wasn't being used in the angular code I translated over, I think we should push it to another PR.